index.html

```diff
@@ -1,16 +1,105 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/normalize.css">
+    <!-- Add a link to your CSS file here (use the line above to guide you) -->
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header class="header-wrapper">
+        <div class="logo">
+            <img class="logo" src="https://freedesignfile.com/upload/2017/08/Cake-Shop-Logo-Vector.jpg" alt="Logo Image">
+        </div>
+          <div class="btn">
+              <button>
+                Contact
+
+              </button>
+            
+            </div>
+    </header>
+    <section>
```

Comment:
You probably don't need this section here as it just wraps the nav and doesn't add any styling